packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts

```diff
@@ -73,13 +74,13 @@ export abstract class InstrumentationBase<T = any>
       return exports;
     }
 
+    const packageJsonPath = path.join(baseDir, 'package.json');
     // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const version = require(path.join(baseDir, 'package.json')).version;
+    const version = existsSync(packageJsonPath) ? require(packageJsonPath).version : undefined;
```

Comment:
even if a file exists it may be not readable (e.g. access rights). What about using a try/catch block instead?

Reply:
Done 👍